From f28546b3943f31333c5e0459aa07536b1d49ef8f Mon Sep 17 00:00:00 2001
From: Jan Janssen <janssen@mpie.de>
Date: Tue, 28 May 2024 11:28:48 +0200
Subject: [PATCH] expect two node tests to fail

---
 t/t0002-basic.t      | 2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/t/t0002-basic.t b/t/t0002-basic.t
index b716715..05b770d 100755
--- a/t/t0002-basic.t
+++ b/t/t0002-basic.t
@@ -108,7 +108,7 @@ test_expect_success 'pmix.job.napps is set to 1' '
 		${GETKEY} --proc=* pmix.job.napps >pmix.job.napps.out &&
 	test_cmp pmix.job.napps.exp pmix.job.napps.out
 '
-test_expect_success 'pmix.nsdir is NOT set' '
+test_expect_failure 'pmix.nsdir is NOT set' '
 	test_must_fail flux run \
 		${GETKEY} --proc=* pmix.nsdir 2>pmix.nsdir.err &&
 	grep NOT-FOUND pmix.nsdir.err
